src/main/resources/log4j2.properties

```diff
@@ -1,5 +1,5 @@
 #property.filename = logs
-property.filename = /home/nate/IdeaProjects/covidPulse/logs
+property.filename = /home/devfecta/IdeaProjects/covidPulse/logs
```

Comment:
Not a change on your end, necessarily, but is there a way we can get this to work with profiles, as well? Would be nice if these changes depending on the environment.

Reply:
If I could get the profiles working that could be an option.